src/main/java/org/owasp/benchmark/score/BenchmarkScore.java

```diff
@@ -786,7 +812,7 @@ else if ( filename.endsWith( ".fpr" ) ) {
 
 		// .fpr files are really .zip files. So we have to extract the .fvdl file out of it to process it
 		    Path path = Paths.get(fileToParse.getPath());
-		    FileSystem fileSystem = FileSystems.newFileSystem(path, null);
+		    FileSystem fileSystem = FileSystems.newFileSystem(path, (java.lang.ClassLoader) null);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *fileSystem*. Currently, there are execution paths that do not contain closure statements. Either a) close *fileSystem* in a try-finally block or b) close the resource by declaring *fileSystem* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).